ui/src/app/workflows/components/submit-workflow-panel.tsx

```diff
@@ -1,4 +1,5 @@
 import {Select} from 'argo-ui/src/components/select/select';
+import {TextInput} from '../../shared/components/text-input';
```

Comment:
This should be ordered below with the other `shared/components` imports

Reply:
Okay, done in fccd30813004c5564e7dd82b67639df8e0f0ade6